src/mappings/entry-point.ts

```diff
@@ -0,0 +1,29 @@
+import { Address } from '@graphprotocol/graph-ts'
+import { AccountDeployed } from '../../generated/EntryPoint/EntryPoint'
+import { EntryPoint, SmartAccount } from '../../generated/schema'
+import { ENTRY_POINT } from '../utils/config'
+import { ADDRESS_ZERO, BIGINT_ZERO } from '../utils/constants'
+
+export function handleAccountDeployed(event: AccountDeployed): void {
```

Comment:
At this point we're assuming all Odyssey wallets we'll be created through the `Entrypoint` contract (i.e. without calling kernel `Factory` directly), is that right?

Reply:
Correct, I have this setup based few wallets creations that I have observed in Odyssey.

Reply:
@mbonfoster could you double check it please?